ui/app/components/section-tabs.hbs

```diff
@@ -3,10 +3,10 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-{{#let (tabs-for-auth-section this.model this.tabType this.paths) as |tabs|}}
+{{#let (tabs-for-auth-section @model this.tabType @paths) as |tabs|}}
   {{#if tabs.length}}
     <div class="tabs-container box is-sideless is-fullwidth is-paddingless is-marginless">
-      <nav class="tabs" aria-label="navigation to manage {{this.model.type}}">
+      <nav class="tabs" aria-label="navigation to manage {{@model.type}}">
         <ul>
           {{#each tabs as |tab|}}
             <li>
```

Comment:
nit: I've been removing `={{true}}` to update test selectors as I see them, not necessary but mentioning in case you want to include!